extensions/2.0/Vendor/EXT_feature_metadata/schema/enum.schema.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "Enum",
+    "type": "object",
+    "description": "An object to define the data type and values for the enum",
+    "allOf": [{ "$ref": "glTFProperty.schema.json" }],
+    "properties": {
+        "valueType": {
+            "type": "string",
+            "enum": [
+                "INT8",
+                "UINT8",
+                "INT16",
+                "UINT16",
+                "INT32",
+                "UINT32",
+                "INT64",
+                "UINT64"
+            ],
+            "description": "The data type for the enum."
+        },
+        "values": {
+            "type": "object",
+            "description": "A dictionary, where each key is an enum name and each value is an enum value. All values must be of the same type as declared in the `valueType`. TODO: should STRING values be allowed for valueType so that this is more readable for JSON property encoding, e.g. for 3D Tiles layers?",
```

Comment:
@lilleyse since we have the PR I'd prefer to see TODOs as a comment on the PR rather than in the schema itself.

As for me, I like the idea for allowing string-valued enums. More readable that way. If performance is a concern, that's what the ints are for.